git-host-info.js

```diff
@@ -25,7 +25,7 @@ var gitHosts = module.exports = {
     'bugstemplate': 'https://{domain}/{user}/{project}/issues',
     'httpstemplate': 'git+https://{auth@}{domain}/{user}/{projectPath}.git{#committish}',
     'tarballtemplate': 'https://{domain}/{user}/{project}/repository/archive.tar.gz?ref={committish}',
-    'pathmatch': /^[/]([^/]+)[/](.+?)(?:[.]git|[/])?$/
+    'pathmatch': /^[/]([^/]+)[/](?!.*\/-\/)(.+?)(?:[.]git|[/])?$/
```

Comment:
I think this will also incorrectly match URLs like `https://gitlab.com/user/foo/bar/baz/repository/archive.tar.gz`